wordpress-mobile/WordPress-iOS
# [Domain Management] Domain Choice UI

Fixes #21775 

## Testing Steps
Go to `PurchaseDomainView.swift` and check the Preview against the designs.
Note: For Previews to function, WordPressAppDelegate launch functions must be commented out.

Functionality will follow once the flow is connected.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
